cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -530,7 +530,7 @@ protected HttpResponse deploy(Class<?> application, int expectedCode, @Nullable
   }
 
   protected HttpResponse deploy(Id.Application appId,
-                                AppRequest<? extends Config> appRequest) throws Exception {
+                                AppRequest<?> appRequest) throws Exception {
```

Comment:
This might break unit test as some test might depend on the config, is this required for your unit test. If not lets not make this change

Reply:
This is needed for my test since I need to test with a configuration of type `String` which doesnt work with `? extends Config`. I'm running all other tests that use this base to verify it didnt break them

Reply:
ok, let's just ensure the bamboo build passes.